app/assets/javascripts/models/ui/reverb_retirement.coffee

```diff
@@ -0,0 +1,19 @@
+#= require util/js.cookies
+
+ns = @edsc.models.ui
+data = @edsc.models.data
+
+ns.ReverbRetirement = do (ko) ->
+  class ReverbRetirement
+    constructor: ->
+
+    returnToReverb: () =>
+      Cookies.set('ReadyForReverbRetirement', 'false', { expires: 7 })
```

Comment:
The cookie expiration should be longer than 7 days. It should expire after Reverb retires.